README.md

```diff
@@ -114,6 +114,32 @@ You can quickly create a Price with the Stripe CLI like so:
 stripe prices create --unit-amount 500 --currency usd -d "product_data[name]=demo"
 ```
 
+<details>
+<summary>With Stripe Tax</summary>
+
```

Comment:
Mabye add a short description of what Stripe Tax is?

Reply:
cd
@bungfut.slack